adafruit_midi/note_off.py

```diff
@@ -37,6 +37,7 @@ class NoteOff(MIDIMessage):  # pylint: disable=duplicate-code
 
     def __init__(self, note, velocity=0, *, channel=None):
         self.note = note_parser(note)
+        """ self.note: key, either int (0-127) or string that will be turned off """
```

Comment:
No need to add the prefix. It is determined by position.

```suggestion
        """key, either int (0-127) or string that will be turned off """
```